android/app/src/main/java/fi/fmi/mobileweather/UpdateWidgetJobIntentService.java

```diff
@@ -195,22 +221,44 @@ protected void updateWidgetUsingGeoId(int geoid, int widget_id) {
             version = "";
         }
 
-        String params = "locations="+geoid+"&l="+language+"&version="+version;
+//        String params = "locations="+geoid+"&l="+language+"&version="+version;
 
         // Large widget
 
-        String url =  "https://widget.weatherproof.fi/android/androidwidget.php?"+params;
+        // forecast url
+        String url = "https://data.fmi.fi/fmi-apikey/ff22323b-ac44-486c-887c-3fb6ddf1116c/timeseries?geoid=" +
```

Comment:
Url with apikey also here

Reply:
UpdateWidgetJobIntentService changes can be ignored because that file will be removed soon. :)

Reply:
File removed